stable/rabbitmq/templates/statefulset.yaml

```diff
@@ -44,7 +44,7 @@ spec:
       affinity:
 {{- if .Values.affinity }}
     {{- with .Values.affinity }}
-{{ tpl . $ | indent 8 }}
+{{ tpl (toYaml .) $ | indent 8 }}
```

Comment:
We found a lot of issue with bad-formatted YAML(s) when using this combo: `tpl (toYaml .) $`
We workaround it creating a new macro.

Could you please add this macro to the `_helpers.tpl`?

```
{{/*
Renders a value that contains template.
Usage:
{{ include "rabbitmq.tplValue" (dict "value" .Values.path.to.the.Value "context" $) }}
*/}}
{{- define "rabbitmq.tplValue" -}}
    {{- if typeIs "string" .value }}
        {{- tpl .value .context }}
    {{- else }}
        {{- tpl (.value | toYaml) .context }}
    {{- end }}
{{- end -}}
```

Then, use:

```
        {{- if .Values.affinity }}
        affinity: {{- include "rabbitmq.tplValue" (dict "value" .Values.affinity "context" $) | nindent 8 }}
        {{- end }}
```

Reply:
Indeed. done!